extension/src/test/suite/experiments/model/tree.test.ts

```diff
@@ -114,21 +109,15 @@ suite('Experiments Tree Test Suite', () => {
 
       expect(
         messageSpy,
-        'when there are no experiments selected we dont send checkpoint type plots'
+        "when there are no experiments selected we don't send checkpoint trend plots"
```

Comment:
[F] By default we now give the plots diff fixture instead of nothing. The test only needs to check the match assumption